services/channel/service/.env.template

```diff
@@ -0,0 +1,25 @@
+NODE_ENV=development
+SERVICE_ID=channel-service
+LOG_LEVEL=DEBUG
+GRAPHQL_GUI_ENABLED=true
+
+PORT=11800
+
+DATABASE_NAME=channel_service
+DATABASE_OWNER=channel_service_owner
+DATABASE_OWNER_PASSWORD=channel_service_owner_pwd
+
+DATABASE_LOGIN=channel_service_login
+DATABASE_LOGIN_PASSWORD=channel_service_login_pwd
+DATABASE_GQL_ROLE=channel_service_gql_role
+
+VIDEO_SERVICE_BASE_URL=https://video.service.eu.axinom.net
+IMAGE_SERVICE_BASE_URL=https://image.service.eu.axinom.net
+LOCALIZATION_SERVICE_BASE_URL=https://localization.service.eu.axinom.net
+
+SERVICE_ACCOUNT_CLIENT_ID=####
+SERVICE_ACCOUNT_CLIENT_SECRET=####
+
+IS_DRM_ENABLED=FALSE
```

Comment:
Do we still need this flag, considering DRM protection boolean field si defined
on the Channel level?